src/ui/base-page-storyEdit.js

```diff
@@ -109,30 +115,17 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                 namespace: "hideEditorShowPreviewer"
             }],
             "{storyEditor}.events.onReadyToBind": {
-                funcName: "sjrk.storyTelling.base.page.storyEdit.loadStoryFromAutosave",
-                args: [
-                    "{that}.options.pageSetup.storyAutosaveKey",
-                    "{storyEditor}",
-                    "", // it replaces the whole story model
-                    "{that}.options.pageSetup.storyAutoloadSourceName"
-                ],
+                funcName: "{that}.initializeStory",
                 priority: "first",
-                namespace: "loadStoryFromAutosave"
+                namespace: "initializeStory"
             },
             "{storyPreviewer}.events.onStoryViewerPreviousRequested": {
                 func: "{that}.showEditorHidePreviewer",
                 args: [true],
                 namespace: "showEditorHidePreviewer"
             },
-            "onStoryShareRequested.submitStory": {
-                funcName: "sjrk.storyTelling.base.page.storyEdit.submitStory",
-                args: [
-                    "{storyEditor}.dom.storyEditorForm",
-                    "{storyPreviewer}.story.model",
-                    "{that}.events.onStoryShareComplete",
-                    "{that}.options.pageSetup.storyAutosaveKey"
-                ]
-            },
+            "onStorySaveToServerRequested.saveStoryToServer": "{that}.saveStoryToServer",
```

Comment:
`onStorySaveToServerRequested` to save the auto save to the server hasn't been called in this pull request. I guess it'll be in the later commits.

Reply:
I added these three events together, but my plan for each of them has changed a little. I don't anticipate using `onStorySaveToServerRequested` in my current work for this Jira. Should I remove it, in that case?

Reply:
I think it's a good idea to remove it and only add when it'll be used. Thanks.